<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MctHookDialog</class>
 <widget class="QDialog" name="MctHookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通话回应</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frameVideoShow">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>551</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHangUp">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInfo">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>对讲来源：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEmergency">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>紧急求助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHelp">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCallCenter">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>呼叫中心</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>音量调节</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderVolume">
   <property name="geometry">
    <rect>
     <x>649</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetVideoChannel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVolumeSub">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVolumeAdd">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>771</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(128, 128,128, 128);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>frameVideoShow</zorder>
  <zorder>pushButtonHangUp</zorder>
  <zorder>lineEditInfo</zorder>
  <zorder>label</zorder>
  <zorder>pushButtonReturn</zorder>
  <zorder>pushButtonEmergency</zorder>
  <zorder>pushButtonHelp</zorder>
  <zorder>pushButtonCallCenter</zorder>
  <zorder>label_2</zorder>
  <zorder>horizontalSliderVolume</zorder>
  <zorder>tableWidgetVideoChannel</zorder>
  <zorder>pushButtonVolumeSub</zorder>
  <zorder>pushButtonVolumeAdd</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditInfo</tabstop>
  <tabstop>pushButtonHangUp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
